### ISSUE TITLE
Can't start sway from LightDM Arch Linux

### ISSUE THREAD

**Issue by Sandmonster** [SirCmpwn/sway#1159]
I want to use sway, it works from i3, it works from tty, but I can't start it from LightDM. I tried to start Weston from LightDM and it worked fine, but when I try to start sway from LightDM screen goes black for sometime and then returns to LightDM. So, I need help to fix this.

**Comment by SirCmpwn** [comment]
Display managers are generally unsupported because none of the significant contributors use one. It's up to you to debug this and figure it out.

**Comment by Sandmonster** [comment]
> Display managers are generally unsupported

OK, I'll keep this issue opened till I figure out what's the problem and how to make it work.

**Comment by Sandmonster** [comment]
Well, that was confusing, in my mind I was absolutely sure that I use SDDM and I was trying to find solution where it couldn't be, then I started exploring log files and there were no logs of sddm, but I noticed lightdm folder and after some time I suddenly realized that I use LightDM. So, now it explains everything LightDM can't start wayland sessions and that's was the problem. Sorry for opening this meaningless issue.
